trails-viz-api/trailsvizapi/config/app_config.py

```diff
@@ -50,4 +51,5 @@
     # 'MBS_GFA': ['flickr', 'twitter', 'instag', 'wta', 'alltrails']
     'EastCascades': ['estimate', 'flickr', 'twitter', 'alltrails', 'ebird', 'wta', 'onsite'],
     'Coronado': ['estimate', 'flickr', 'twitter', 'alltrails', 'ebird', 'onsite']
```

Comment:
The `'Coronado'` line also needs a comma after it if you are adding a new line after (each line should end with a comma).